deploy/operator.yaml

```diff
@@ -41,6 +41,8 @@ spec:
               value: "http://localhost:6385/v1/"
             - name: IRONIC_INSPECTOR_ENDPOINT
               value: "http://localhost:5050/v1/"
+            - name: OPERATOR_CONCURRENCY
+              value: 3
```

Comment:
Same thing.

Reply:
You're correct. Its not needed. Going to remove it.